bin/generate

```diff
@@ -124,7 +124,7 @@ def add_requirements(text, meta, defaults):
     text.append(section_header('Dependencies', 2))
     dependencies = []
     if len(meta['dependencies']) > 0:
-        dependencies = ['- {0}'.format(dependency) for dependency in defaults['dependencies']]
+        dependencies = ['- {0}'.format(dependency) for dependency in meta['dependencies']]
```

Comment:
Can we append ` ansible role` to the string here so that the readme has the right text?

Reply:
Sure.